cypress/e2e/CMS/$Biography/index.js

```diff
@@ -0,0 +1,146 @@
+        cy.get('.profile-panel-image img').then($el=>{
+        const source = $el[0].getAttribute('src');
+        const actSrc = source.replace(/\?itok=[\S]+/,'').replace(/^(.*?)\/public/,'')
+        const expectedSrc = (imageSrc.replace(/\?itok=[\S]+/,'')).replace(/^(.*?)\/public/,'')
+        expect(actSrc).to.include(expectedSrc);
+        })
+        })
+
+
+And('the individual {int} title is displayed as {string}',(num,option1) =>{
+    cy.get(`#nvcgSlProfilePanel div:contains("${option1}")`).eq(num).should('be.visible')
+
+})
+And('the individual {int} title is displayed as {string}',(num,option2) =>{
+      cy.get(`#nvcgSlProfilePanel div:contains("${option2}")`).eq(num).should('be.visible')
+    
+})
+And('the following organizations with links are listed for this individual',(dataTable)=> {
+    for (const { organization, url } of dataTable.hashes()) {
+        cy.get(`#nvcgSlProfilePanel a:contains('${organization}')`).should('have.attr', 'href', url)
+    }
+})
+
+And('individuals office is at {string}',(address) =>{
+    cy.document().then(doc => {
+        const addressInnerText = doc.querySelector("div[class='profile-panel-content'] p").innerText;
+        expect(addressInnerText).to.be.eq(address)
+        cy.get("div[class='profile-panel-content'] p").should('be.visible')
+    });
+})
+
+And('contact phone number is {string}',(phone) =>{
+    cy.get('.profile-contact-info').should('be.visible').and('include.text', phone)
+})
+And('contact email is {string}',(email) =>{
+    cy.get(`a:contains("${email}")`).should('be.visible').and('include.text',email)
+
+})
+And('the following social media links are present', (dataTable) => {
+    for (const { socialMedia, href } of dataTable.hashes()) {  
+    cy.get(`a[href="${href}"]`).should('be.visible')
+      
+    }
+})
+
+And('user removes the Biography Image',() =>{
+    cy.get('span:contains("Biography Image")').parent().as('imageUpload').click();
+    cy.get('#edit-field-image-promotional-current-items-0-remove-button').click({force: true})
+ 
+})
+
+let imageSrc1;
+And('user selects {int} Biography Image from the list of images', (number) => {
+    cy.get('span:contains("Biography Image")').parent().as('imageUpload').click();
+    cy.get('input[value="Select Image"]').click({ force: true });
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id^='edit-entity-browser-select-media6']").eq(number -2).check();
+  
+    cy.getIframeBody('iframe.entity-browser-modal-iframe').find("input[id='edit-submit'][value='Select image']").click({ force: true });
+});
+    And('user remembers the source of selected biography image for further verification', () => {
+    cy.get('details img').then($el => {
+        imageSrc1 = $el[0].getAttribute('src');
+    })
+})
+
+ And('user clicks on title {string} from the list of content',(title) =>{
+    cy.get("a[href='/about-cancer/understanding/biography']").should('have.text',title).click({force: true})     
```

Comment:
1. This step already been implemented in common - you should not be duplicating things 
2.  Hardcoded url????